# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../vcd/no_install_vbasedir.patch
# Copyright (C) 2008 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- trunk/src/vcd/Makefile.am.orig	2000-01-01 00:00:00.000000000 +0200
+++ trunk/src/vcd/Makefile.am	2008-02-01 08:29:51.000000000 +0200
@@ -55,6 +55,5 @@
 install-data-local:
 	$(install_sh)    -m 600 $(srcdir)/vcd.conf $(DESTDIR)$(sysconfdir)/vcd.conf
 	$(mkinstalldirs) -m 755 $(DESTDIR)$(localstatedir)/vcd
-	$(mkinstalldirs) -m 000 $(DESTDIR)$(vbasedir)
 
 endif
